beaker-vmware.gemspec

```diff
@@ -20,25 +20,11 @@ Gem::Specification.new do |s|
   # Testing dependencies
   s.add_development_dependency 'rspec', '~> 3.0'
   s.add_development_dependency 'rspec-its'
-  if RUBY_VERSION < "2.3"
-    s.add_development_dependency 'fakefs', '~> 0.6', '< 0.14'
-  else
-    s.add_development_dependency 'fakefs', '~> 0.6'
-  end
-  s.add_development_dependency 'rake', '~> 10.1'
-  s.add_development_dependency 'simplecov'
-  s.add_development_dependency 'pry', '~> 0.10'
-  s.add_development_dependency 'beaker', '~> 3.0'
-
-  # Documentation dependencies
-  s.add_development_dependency 'yard'
-  s.add_development_dependency 'markdown'
-  s.add_development_dependency 'thin'
```

Comment:
If you're dropping this, you should also remove any related things from `Rakefile`.

Reply:
I added them back, in case we ever want to generate and publish the docs again